examples/addsvc/main.go

```diff
@@ -123,35 +131,30 @@ func main() {
 	go func() {
 		var (
 			transportLogger = log.NewContext(logger).With("transport", "HTTP/JSON")
-			tracingLogger   = log.NewContext(transportLogger).With("component", "tracing")
-			newSumSpan      = zipkin.MakeNewSpanFunc(*zipkinHostPort, *zipkinServiceName, "sum")
```

Comment:
note that OpenTracing doesn't need these sorts of `new*Span` functions... that's all part of the `Tracer` impl.